tsconfig.json

```diff
@@ -10,16 +10,13 @@
     "skipLibCheck": true,
     "strict": false,
     "forceConsistentCasingInFileNames": true,
-    "noEmit": true,
+    "noEmit": false,
```

Comment:
Had to switch this to `false` or else I think `ts-loader` complains trying to compile some `.ts` files without any actual TypeScript in them